docs/netserver/config/timezone.md

```diff
@@ -22,3 +22,8 @@ Configuration values related to TimeZone.
 | Name | Description |
 |---|---|
 | ServiceUrl | URL for the TimeZone service. |
+
+See the [NetServer Core reference][1] for details about handling this programmatically.
+
+<!-- Referenced links -->
+[1]: <xref:SuperOffice.Configuration.ConfigFile.Timezone>
```

Comment:
This xref link does not work
Try: [1]: xref:SuperOffice.Configuration.ConfigFile.TimeZone
or inline it:
[NetServer Core reference](xref:SuperOffice.Configuration.ConfigFile.TimeZone)